src/core/Base.ts

```diff
@@ -150,10 +150,6 @@ export class Base {
         await player.saveToDatabase();
         if (disconenctAll) {
           player.disconnect("now");
-        } else {
-          player.every((p) => {
-            p.send(Variant.from("OnConsoleMessage", "`4System: `oSaving Players Data"), TextPacket.from(PacketTypes.ACTION, "action|play_sfx", `file|audio/beep.wav`, `delayMS|0`));
```

Comment:
Just remove the nested loop, instead add this, since that loop broadcasted into every player might resulting a duplicate OnConsoleMessage.

do this instead
```ts
player.send(Variant.from("OnConsoleMessage", "`4System: `oSaving Players Data"), TextPacket.from(PacketTypes.ACTION, "action|play_sfx", `file|audio/beep.wav`, `delayMS|0`));	
```

Reply:
Just deleted the OnConsoleMessage on the new commit, just realized that the saveworld function does not broadcast it's saving to the players.